src/string.cr

```diff
@@ -509,7 +541,7 @@ class String
     end
   end
 
-  private def to_u64_info(base, whitespace, underscore, prefix, strict)
+  private def to_u128_info(base, whitespace, underscore, prefix, strict)
```

Comment:
I'm concerned about all `to_i` calls having to go through 128bit mathematics: it probably slows down the conversion process on processors without 128bit maths support (i.e. most of them).

I think it's a good idea to duplicate this method, and have both `to_u64_info` and `to_u128_info`. This can probably be done quite easily without extra boilerplate with macros.

Reply:
Yes, I agree. Also two test are failed because I extend to_u64_info method to support 128 bit integer and now they works incorrect with 64 bit.
I will fix it.